server/server.go

```diff
@@ -969,7 +969,8 @@ func (s *Server) canEval(ctx context.Context) bool {
 	return false
 }
 
-func (s *Server) bundlesReady(pluginStatuses map[string]*plugins.Status) bool {
+// BundlesReady checks if the configured bundles are ready
+func (s *Server) BundlesReady(pluginStatuses map[string]*plugins.Status) bool {
```

Comment:
To implement the check we'll likely want to refactor `server.Server#unversionedGetHealth` to pull out the logic to check on plugin status(es) too. Similar to the `BundlesReady` we can have a `PluginsReady` or similar.

Looking at the current implementation it might make the most sense to move this (and the plugin one) over to the `runtime` instead of as public methods on the server. The same functionality could/should be added to the REPL, so having them on the runtime makes the most sense IMO. It doesn't look like there is anything specific to the server in the implementation of `BundlesReady` and shouldn't be for the `PluginsReady` too.